### PR TITLE
access 0.10.9 (new formula)

### PR DESCRIPTION
**Pull request by ethernetdan**
The `access` tool allows permissions to be easily requested and granted so systems can be more secure without losing productivity.

- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/access.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    system "go", "build", *std_go_args(ldflags: ldflags), "./cmd/access"
+
+    # Install shell completions
+    generate_completions_from_executable(bin/"access", "completion")
+  end
+
+  test do
+    test_file = testpath/"access.yaml"
+    test_file.write("")
```
Comment (carlocab) [inline Formula/access.rb:27]:
```suggestion
    touch test_file
```

File: Formula/access.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    Dir.chdir(testpath) do
+      system bin/"access", "config", "set", "space", "some-space"
+    end
```
Comment (carlocab) [inline Formula/access.rb:30]: `testpath` is already the working directory.
```suggestion
    system bin/"access", "config", "set", "space", "some-space"
```

File: Formula/access.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    assert_equal "space: some-space", File.read(test_file).strip
```
Comment (carlocab) [inline Formula/access.rb:31]: `test_file` is a `Pathname`, so we can use it like one:
```suggestion
    assert_equal "space: some-space", test_file.read.strip
```

**Review by carlocab — COMMENTED** [review]
Thanks for the PR! A few suggestions.

File: Formula/access.rb (hunk 3 above)
Comment (ethernetdan) [reply Formula/access.rb]: Oh that's neat

**Comment by ethernetdan** [comment]
Great suggestions @carlocab! Updated to include all of them

**Comment by carlocab** [comment]
Thanks; could you squash your commits together so we can get rid of the https://github.com/Homebrew/homebrew-core/labels/autosquash label?

**Review by carlocab — APPROVED** [review]
Thanks!

**Comment by ethernetdan** [comment]
@carlocab thanks for maintaining homebrew! Sorry about not squashing, I misunderstood the mechanics of `autosquash`

**Review by ZhongRuoyu — APPROVED** [review]
Thanks for your contribution, @ethernetdan!

**Pull request merged**